sdk/cosmosdb/cosmos/test/internal/unit/sasToken.spec.ts

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import assert from "assert";
+import { CosmosClient } from "../../../dist-esm/";
+import { endpoint, masterKey } from "../../public/common/_testConfig";
+import { SasTokenPermissionKind } from "../../../dist-esm/common";
+import { createAuthorizationSasToken } from "../../../dist-esm/utils/SasToken"
+import { SasTokenProperties } from "../../../dist-esm/client/SasToken/SasTokenProperties";
+
+describe.only("SAS Token Authorization", function () {
```

Comment:
```suggestion
describe.skip("SAS Token Authorization", function () {
```
Need to remove only. But also lets just skip this for now since this test won't pass against the public emulator

Reply:
Also add a comment why this is skipped to remind our future selves